var/spack/repos/builtin/packages/root/package.py

```diff
@@ -392,12 +392,11 @@ def cmake_args(self):
             define_from_variant('qtgsi', 'qt4'),  # See conflicts
             define_from_variant('r'),
             define('rfio', False),
-            define('roofit', False),
+            define_from_variant('roofit', 'roofit'),
             define_from_variant('root7'),  # See conflicts
             define('ruby', False),
             define('sapdb', False),
             define_from_variant('shadowpw', 'shadow'),
-            define_from_variant('spectrum'),
```

Comment:
Why is this one going away?

Reply:
oha, I think I was changing an old version of this file and not looking before commiting. Thanks!